modules/c3-assisted-installer-completing-installation-nodes.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_oci/installing-c3-assisted-installer.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="c3-ai-completing-installation-nodes_{context}"]
+= Assigning node roles
+
+Delete the disconnected hosts and assign roles to the remaining hosts. Rename any hosts with names shorter than 63 characters.
+
+.Procedure
+
+. From the link:https://console.redhat.com/openshift/assisted-installer/clusters[{ai-full} web console], select the cluster and navigate to the *Host discovery* page. 
++
+If the Terraform scripts completed successfully, twelve hosts are listed on the page. Six hosts have the status *Disconnected* caused when you restarted the isntance on C3.
+
+. Delete the six hosts with a *Disconnected* status, by clicking the option button for the host and selecting *Remove host*. The status of the remaining hosts changes from *Insufficient* to *Ready*. This can take up to three minutes.
+
+. From the *Role* column, assign the three nodes with a boot size of 1TB the *Control plane* role, and the three nodes with boot size 100GB the *Worker* role. 
```

Comment:
assign the three nodes with a boot size of 1TB the *Control plane* ?  you mean here to boot disk size ,  why its 1TB ?  or 100GB , if it depends on  customer hardware it may be different

Reply:
It was in the Oracle document.